clients/instance/ibm-pi-dhcp.go

```diff
@@ -8,28 +8,30 @@ import (
 	"github.com/IBM-Cloud/power-go-client/helpers"
 
 	"github.com/IBM-Cloud/power-go-client/ibmpisession"
-	"github.com/IBM-Cloud/power-go-client/power/client/p_cloud_service_d_h_c_p"
+	p "github.com/IBM-Cloud/power-go-client/power/client/p_cloud_service_d_h_c_p"
```

Comment:
Could there be a more descriptive name than 'p'?

Reply:
I think it is go naming convention to use the initials instead of long names.